Source/Protos/ni/measurementlink/pinmap/v1/pin_map_service.proto

```diff
@@ -233,3 +236,8 @@ message ChannelMapping {
     // User-defined identifier for the multiplexer type in the pin map editor.
     string multiplexer_type_id = 6;
 }
+
+message ResolvedPinsOrRelays {
+  // List of pin or relay names in the pin or relay group.
```

Comment:
Please have 4 spaces.

Reply:
Thanks for the reference, Jason. I have changed the other lines in the pin map and session management proto to be consistent with 2 space indentation.

Reply:
In general, we shouldn't really being modifying/editing any of the proto files as they are copied into this repo. The source of truth should be the upstream ni-apis repo. If there are edits to be made, that is where they should be made. At some point, we should probably try to use a git submodule to incorporate these files into this repo rather than copying them. However, that's effectively tech debt that doesn't need to gate this PR unless you are feeling motivated to try and get that done.

Reply:
ni-apis still isn't quite public. So until then we can't use it as a submodule.